src/Application/BaseApplication.php

```diff
@@ -285,7 +285,6 @@ protected function registerTasksForTransferMethod(Workflow $workflow, $transferM
     }
 
     /**
-     * @param \TYPO3\Surf\Domain\Model\Workflow $workflow
```

Comment:
Because the Type Declaration is already indicating the type of the parameter. This cleanup is only a intermediate step towards fully scalar type and return type declarations. So that in the end, we can get rid of all superfluous doc blocks.